packages/admin/src/brevoContacts/BrevoContactsGrid.tsx

```diff
@@ -0,0 +1,202 @@
+import { gql, useApolloClient, useQuery } from "@apollo/client";
+import {
+    MainContent,
+    messages,
+    RowActionsItem,
+    RowActionsMenu,
+    Toolbar,
+    ToolbarFillSpace,
+    ToolbarItem,
+    ToolbarTitleItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import { Block, Check, Delete } from "@comet/admin-icons";
+import { ContentScopeInterface } from "@comet/cms-admin";
+import { DataGrid, GridColDef, GridToolbarQuickFilter } from "@mui/x-data-grid";
+import * as React from "react";
+import { FormattedMessage, IntlShape, useIntl } from "react-intl";
+
+import {
+    GQLBrevoContactsGridQuery,
+    GQLBrevoContactsGridQueryVariables,
+    GQLBrevoContactsListFragment,
+    GQLDeleteBrevoContactMutation,
+    GQLDeleteBrevoContactMutationVariables,
+    GQLUpdateBrevoContactMutation,
+    GQLUpdateBrevoContactMutationVariables,
+} from "./BrevoContactsGrid.generated";
+
+const brevoContactsFragment = gql`
+    fragment BrevoContactsList on BrevoContact {
+        id
+        createdAt
+        modifiedAt
+        email
+        emailBlacklisted
+        smsBlacklisted
+    }
+`;
+
+const brevoContactsQuery = gql`
+    query BrevoContactsGrid($offset: Int, $limit: Int, $email: String, $scope: EmailCampaignContentScopeInput!) {
+        brevoContacts(offset: $offset, limit: $limit, email: $email, scope: $scope) {
+            nodes {
+                ...BrevoContactsList
+            }
+            totalCount
+        }
+    }
+    ${brevoContactsFragment}
+`;
+
+const deleteBrevoContactMutation = gql`
+    mutation DeleteBrevoContact($id: Int!) {
+        deleteBrevoContact(id: $id)
+    }
+`;
+
+const updateBrevoContactMutation = gql`
+    mutation UpdateBrevoContact($id: Int!, $input: BrevoContactUpdateInput!) {
+        updateBrevoContact(id: $id, input: $input) {
+            id
+        }
+    }
+`;
+
+function BrevoContactsGridToolbar({ intl }: { intl: IntlShape }) {
+    return (
+        <Toolbar>
+            <ToolbarTitleItem>
+                <FormattedMessage id="comet.brevoContact.title" defaultMessage="Contacts" />
+            </ToolbarTitleItem>
+            <ToolbarItem>
+                <GridToolbarQuickFilter
+                    placeholder={intl.formatMessage({ id: "comet.brevoContact.search", defaultMessage: "Search email address" })}
+                />
+            </ToolbarItem>
+            <ToolbarFillSpace />
+        </Toolbar>
+    );
+}
+
+export function BrevoContactsGrid({ scope }: { scope: ContentScopeInterface }): React.ReactElement {
+    const client = useApolloClient();
+    const intl = useIntl();
+    const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("BrevoContactsGrid") };
+
+    const columns: GridColDef<GQLBrevoContactsListFragment>[] = [
+        {
+            field: "createdAt",
+            headerName: intl.formatMessage({ id: "comet.brevoContact.subscribedAt", defaultMessage: "Subscribed At" }),
```

Comment:
usually the comet prefix is reserved for comet directly. Since this is a comet module, it could be okay but I'd still suggest another prefix. At some point we could define a key in the same way as in main comet library, and then we would have the same key twice with a different default message, which could lead to confusion at some point (because afaik it would overwrite one defaultMessage then)

So I'd suggest brevoModule.brevoContact, or do you have a better naming suggestion?

Reply:
I changed it to `cometBrevoModule` in [853a758](https://github.com/vivid-planet/comet-brevo-module/pull/20/commits/853a75879e572e8e9e623d2ee1c4f26455666410)